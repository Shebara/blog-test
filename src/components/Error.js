import React from 'react';

const Error = ({title, description}) => {
    return (
        <div className="Error">
            <h1>{title}</h1>
            <div>{description}</div>
        </div>
    );
}

export default Error;