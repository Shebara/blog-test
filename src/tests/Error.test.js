import React from 'react';
import { shallow } from 'enzyme';
import Error from '../components/Error';

describe('Error', () => {
    it('should render without errors', () => {
        const component = shallow(<Error />);

        expect(component.getElements()).toMatchSnapshot();
    });
});
