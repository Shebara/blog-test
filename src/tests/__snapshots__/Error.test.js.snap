// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Error should render without errors 1`] = `
Array [
  <div
    className="Error"
  >
    <h1 />
    <div />
  </div>,
]
`;
